_haikus/chance_haiku.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Chance's Haiku
+author:Chance
```

Comment:
Add a space between the colon and your name, like so:

```
author: Chance
```

Reply:
Fixed